api/envoy/config/filter/network/thrift_proxy/v2alpha1/route.proto

```diff
@@ -82,6 +82,12 @@ message RouteAction {
     // request is routed to one of the upstream clusters based on weights
     // assigned to each cluster.
     WeightedCluster weighted_clusters = 2;
+
+    // Envoy will determine the cluster to route to by reading the value of the
+    // Thrift header named by cluster_header from the request headers. If the
+    // header is not found or the referenced cluster does not exist, Envoy will
+    // return an exception.
```

Comment:
What does returning an exception mean from an API perspective? This seems alarming behavior 😨

Reply:
In thrift land it's the equivalent of a 503... I tried to be consistent with:

https://github.com/envoyproxy/envoy/blob/master/api/envoy/api/v2/route/route.proto#L670

Shall I say a "thrift exception" ?

Reply:
Or even better, a "thrift response exception"?

Reply:
Improved the wording a bit.

Reply:
I see, Thrift has a notion of exception messages. LGTM, thanks.